ra2ce/analysis/analysis_config_data/analysis_config_data.py

```diff
@@ -237,7 +237,7 @@ def adaptation(self) -> AnalysisSectionAdaptation | None:
 
     def get_analysis(
         self, analysis: AnalysisEnum | AnalysisDamagesEnum | AnalysisLossesEnum
-    ) -> AnalysisSectionBase | None:
+    ) -> AnalysisSectionDamages | AnalysisSectionLosses | AnalysisSectionAdaptation | None:
```

Comment:
create an enum or a self-defined type to avoid this. ( `ANALYSIS_TYPE = AnalysisSectionDamages | AnalysisSectionLosses | AnalysisSectionAdaptation`)

Reply:
Or create a protocol?

Reply:
Created a simple protocol to keep things clean

Reply:
Done